src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -65,6 +65,23 @@
   protected String presentLeaderboardSelector = "div[id*='TOP_LEADERBOARD']";
   protected String presentHighImpactSlotSelector = "div[id*='INVISIBLE_HIGH_IMPACT']";
 
+  protected String
+      presentFandomDesktopArticleTopLeaderboardSelector = "div[id*='gpt-top-leaderboard-desktop']";
+  protected String
+      presentFandomDesktopArticleBottomLeaderboardSelector = "div[id*='gpt-bottom-leaderboard-desktop']";
+  protected String
+      presentFandomMobileArticleTopLeaderboardSelector = "div[id*='gpt-top-leaderboard-mobile']";
+  protected String
+      presentFandomMobileArticleBottomLeaderboardSelector = "div[id*='gpt-bottom-leaderboard-mobile']";
+  protected String
+      presentFandomHubTopLeaderboardSelector = "div[id*='gpt-top-leaderboard']";
+  protected String
+      presentFandomHubBottomLeaderboardSelector = "div[id*='gpt-bottom-leaderboard']";
+
+  protected String presentFandomTopBoxadSelector = "div[id*='gpt-top-boxad']";
+  protected String presentFandomBottomBoxadSelector = "div[id*='gpt-bottom-boxad']";
+
+
```

Comment:
I decided to not implement this now - number of slot on Fandom pages is more limited and problem is more with checking if its desktop or mobile version. Also I moved all Fandom slots to AdsFandomObject so they are now in one place.

I added this comment to ADEN-3496 to be considered later.